src/porepy/applications/complete_setups/mandel_biot.py

```diff
@@ -1284,7 +1245,10 @@ class MandelGeometry(pp.ModelGeometry):
     """Simulation model parameters."""
 
     fracture_network: pp.FractureNetwork2d
-    """Fracture network. Empty in this case."""
+    """Two-dimensional fracture network object. Set by a mixin instance of
```

Comment:
Are you sure this is needed for the class which assigns it?

Reply:
Good catch. This is not needed. Fixed.